ibm/mas_devops/roles/cp4d_wds/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+wdschannel: v4.0
+wdsversion: 4.0.2
+wdsstorageclass: ocs-storagecluster-ceph-rbd
```

Comment:
We can't build in a default for storage classes, as it won't work unless that class is available in the cluster